opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -201,7 +201,7 @@ def shutdown(self):
         result = True
 
         for metric_reader in self._sdk_config.metric_readers:
-            result = result and metric_reader.shutdown()
+            result = metric_reader.shutdown() and result
```

Comment:
How does this work.... if shutdown fails on the first metric reader, wont a warning be logged for every subsequent reader shutdown?

Reply:
Right, fixed :v: